src/pages/office/components/fault-comment/fault-comment.ts

```diff
@@ -24,15 +31,21 @@ export class FaultCommentComponent implements OnChanges {
   @Input()
   faultType: string;
 
+  @Input()
+  faultCount: number;
   @Output()
   faultCommentChange = new EventEmitter<CommentedCompetency>();
 
   static readonly fieldName: string = 'faultComment';
   constructor(private outcomeBehaviourProvider: OutcomeBehaviourMapProvider) { }
 
   ngOnChanges(): void {
-    if (this.outcomeBehaviourProvider.getVisibilityType(this.outcome, FaultCommentComponent.fieldName) ===
-      VisibilityType.NotVisible) {
+    const fieldVisibility = this.outcomeBehaviourProvider.getVisibilityType(
+      this.outcome, FaultCommentComponent.fieldName);
+
+    if (fieldVisibility === VisibilityType.NotVisible ||
```

Comment:
It's quite tricky to understand the rationale for this, perhaps a comment?

Reply:
comment added referencing bug and explaining why